coral-component-table/src/scripts/TableRow.js

```diff
@@ -447,7 +447,8 @@ const TableRow = Decorator(class extends BaseComponent(HTMLTableRowElement) {
               .split(/\s+/g)
               .filter(id => selectHandle._elements.id !== id && this._elements.accessibilityState.id !== id)
               .join(' ');
-            selectHandle.labelledBy = selectHandle._elements.id + ' ' + ids;
+            let titleCell = this.querySelector(".foundation-collection-item-title");
```

Comment:
foundation-collection-item-title is not coral selector. you should push this change to ui.content